### PR TITLE
(NFC) CRM-19606 : provide help text for providing mysql port info

### PR DESCRIPTION
**Pull request by sharique**
I'm new to Civicrm, starting with a small patch. 
----------------------------------------
* CRM-19606: Provide help text in installer to provide mysql port, if mysql is running on different port
  https://issues.civicrm.org/jira/browse/CRM-19606

### CODE REVIEW

**Comment by totten** [comment]
jenkins, ok to test

**Comment by jitendrapurohit** [comment]
This seems to be creating a little [mess on the installation page].(https://cloud.githubusercontent.com/assets/5929648/20137520/fa253080-a6a0-11e6-9770-96f7e625c951.png) 

Can we add `testResult` class to this span so that it gets displayed similar to the `seedLanguage` help text displayed above ?

File: install/template.html [hunk 1]
```diff
@@ -98,6 +98,7 @@
 <h4><?php echo ts('CiviCRM Database Settings'); ?></h4>
 <p style="margin-left: 2em" id="mysql_credentials">
   <label for="mysql_server"> <span><?php echo ts('MySQL server:'); ?></span> <input id="mysql_server" type="text" name="mysql[server]" value="<?php echo $databaseConfig['server']; ?>" /></label> <br />
+  <span><?php echo ts('If your mysql server is running on other port then default 3306, provide server info as server:port (i.e. localhost:1234) ') ?> </span> </br>
```
Comment (jitendrapurohit) [inline install/template.html]: I think the message should be `(If your mysql server is running on a port other than default 3306, provide server info as a server:port (i.e. localhost:1234))`.
Note the typo then**.

**Comment by jitendrapurohit** [comment]
Tested the PR. Installation page looks like -

final

**Comment by monishdeb** [comment]
Congrats @sharique for your first PR :)

**Comment by sharique** [comment]
@monishdeb Thanks. Hopefully more to come.

**Pull request merged**